test/kubernetes-test.sh

```diff
@@ -10,10 +10,10 @@
 # Cluster.
 helm template --validate install/kubernetes/cilium \
   --namespace=kube-system \
-  --set image.tag=latest \
-  --set image.repository=k8s1:5000/cilium/cilium-dev \
-  --set operator.image.repository=k8s1:5000/cilium/operator \
-  --set operator.image.tag=latest \
+  --set image.tag=$1 \
+  --set image.repository=quay.io/cilium/cilium-ci \
+  --set operator.image.repository=quay.io/cilium/operator-generic-ci \
+  --set operator.image.tag=$1 \
```

Comment:
This won't work with the current helm template. https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/cilium-operator-deployment.yaml#L161-L167

We will have to do something like below:

```
--set operator.image.repository=quay.io/cilium/operator \
--set operator.image.suffix=-ci
```

Reply:
Good catch!